packages/clis/parser/game-files/sii-visitors.ts

```diff
@@ -101,6 +116,11 @@ class JsonConverterVisitor extends getSiiVisitorClass<
   }
 }
 
+interface NumberTupleChild {
+  value: number | null;
```

Comment:
I wrote down your preference in my notes and will try to remember it next time.

In this particular case though, I think it has to be `null` because
- `null` and `undefined` are distinct concepts in JavaScript, and
- the schema validation works off of JSON types, which don't include `undefined`.

Reply:
Both fair points 👍. Leaving this as `null` SGTM.